build-system/release-tagger/make-release.js

```diff
@@ -138,6 +138,42 @@ function _createComponentSections(prs) {
   return sectionsMarkdown;
 }
 
+/**
+ * Determines if a path is a path to a component directory.
+ * @param {string} path
+ * @param {?string=} name
+ * @param {?string=} version
+ * @return {boolean}
+ */
+function isComponentPath(path, name, version) {
+  const nameDir = name ? `${name}/` : '';
+  const versionDir = version ? `${version}/` : '';
+  return (
+    path.startsWith(`extensions/${nameDir}${versionDir}`) ||
+    path.startsWith(`src/bento/components/${nameDir}${versionDir}`)
+  );
+}
+
+/**
+ * Gets the name of a component from its directory path.
+ * Returns undefined if the path is not a component directory.
+ * @param {string} path
+ * @return {string|undefined}
+ */
+function getComponentNameFromPath(path) {
+  if (!isComponentPath(path)) {
+    return;
+  }
```

Comment:
This isn't needed because if it doesn't start with `extensions/` or `src/bento/components/` it won't return anything anyways